src/inject/dynamic-theme/inline-style.ts

```diff
@@ -317,6 +319,11 @@ export function overrideInlineStyle(element: HTMLElement, theme: FilterConfig, i
         }
         if (overrides.hasOwnProperty(property)) {
             setCustomProp(property, property, value);
+        } else {
+            const customProp = customPropsList[property];
```

Comment:
As far as I understand `customProp` holds real CSS prop here, while `property` is actually a custom property.